src/apps/VirtualApp/VirtualChat.js

```diff
@@ -104,6 +104,12 @@ class VirtualChat extends Component {
             console.log(`[VirtualChat] ${message}`);
             captureMessage(`Chatroom: ${message}`, {source: "Textroom"});
           },
+          slowLink: (uplink, lost, mid) => {
+            const slowLinkType = uplink ? 'sending' : 'receiving';
+            const message = 'Janus reports slow link problems ' + slowLinkType + ' packets on mid ' + mid + ' (' + lost + ' lost packets)';
```

Comment:
I wouldn't have mid and lost in title. you can have it in extra data

Reply:
Fixed, thanks! good one.